packages/fluentui/react-northstar/src/components/Menu/Menu.tsx

```diff
@@ -256,6 +265,22 @@ export const Menu = compose<'ul', MenuProps, MenuStylesProps, {}, {}>(
       });
     };
 
+    const childProps: MenuContextValue = {
+      activeIndex: +activeIndex,
+      onItemClick: handleClick,
+      variables,
+      pointing,
+      primary,
+      underlined,
+      iconOnly,
+      vertical,
+      secondary,
+      pills,
+      inSubmenu: props.submenu,
+      accessibility: accessibility && accessibility(props).childBehaviors,
```

Comment:
I don't think that this object is stable

Reply:
Hm?

Reply:
Every time when `accessibility(props)` will be evaluated it will return new `childBehaviors` => all children will be notified => rerender everything 💣